techmatters/terraso-mobile-client
# feat: soil info sheets

## Description

Implement soil info sheets for temp locations and sites. Refactor some related components for ease of implementation.

### Checklist


### Related Issues
Fixes #1156

### Verification steps

Select a site or temporary location. Go to "explore the data" and press the temporary "(Match Goes Here)" button to open the sheet. Verify compliance with design spec for site and temporary location cases.
